src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -682,7 +682,7 @@ def runSteps(image, config, branchName, axis, steps=config.steps, runtime) {
 def getConfigVal(config, list, defaultVal=null, toString=true, oneStep=null, useTemplate=false) {
     def val = oneStep ?: config
     for (int i=0; i<list.size(); i++) {
-        item = list[i]
+        def item = list[i]
```

Comment:
how and why it solves bug with wrong values?

Reply:
I try to explain it in my PR message, did you read it? is it not clear enough?

Reply:
missed it, thanks, yep, makes perfect sense